File: Client/member-registration-portal/src/app/entity/claim.ts
export class Claim{
    claimId: string = "";
    name: string = "";
    memberName: string = "";
    dob: Date = new Date();
    admissionDate: Date = new Date();
    dischargeDate: Date = new Date();
    provider: string = "";
    totalBill: number = 0;
}

File: Server/MemberService/src/main/java/com/member/models/Claim.java
package com.member.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Claim {
    private String claimId;
    private String name;
    private String memberName;
    private LocalDate dob;
    private LocalDate admissionDate;
    private LocalDate dischargeDate;
    private String provider;
    private double totalBill;
}
